.github/workflows/test-sdks.yml

```diff
@@ -1,4 +1,5 @@
 name: Test SDKs Locally
+name: Test SDKs Locally
```

Comment:
Looks like this was duplicated by accident

Reply:
Thanks for the catch; late night merges...